### DISCUSSION TITLE
Displaying new model attributes in the templates?

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#148] [Q&A] [answered]
I want to show some additional attributes above the video title. I have added the attributes to the files model and modified the views appropriately, but now, how do I bring the attributes into the templates?

I see the title is loaded by _commons.js from something called e[c], but tampering with e[c] in _commons.js hasn't worked for me. How can I get these attributes into the templates (main page, details page, details page sidebar)?

**Comment by mgogoulos** [comment, marked as answer]
Hi, the frontend code is going to be published soon, there's still some work to be done
